node-agent/workload_manager/src/workload_manager/workload/container.rs

```diff
@@ -0,0 +1,127 @@
+use bollard::container::{
+    Config, KillContainerOptions, RemoveContainerOptions, RenameContainerOptions,
+    StopContainerOptions,
+};
+use bollard::Docker;
+
+use bollard::image::CreateImageOptions;
+use futures_util::TryStreamExt;
+
+use super::workload_trait::Workload;
+use proto::agent::Instance;
+
+pub struct Container {
+    id: String,
+}
+
+impl Container {
+    /**
+     * Removes a container
+     */
+    async fn remove(&self) {
+        let docker = Docker::connect_with_socket_defaults().unwrap();
+        docker
+            .remove_container(
+                self.get_id().as_str(),
+                Some(RemoveContainerOptions {
+                    force: true,
+                    ..Default::default()
+                }),
+            )
+            .await
+            .unwrap();
+    }
+}
+
+#[tonic::async_trait]
+impl Workload for Container {
+    fn get_id(&self) -> String {
+        self.id.to_string()
+    }
+
+    /**
+     * Create a new workload and start it
+     */
+    async fn new(instance: Instance) -> Self {
+        let docker = Docker::connect_with_socket_defaults().unwrap();
```

Comment:
You're including a bunch of `unwrap()`. Please try to define and propagate errors properly instead.